config/settings.yml

```diff
@@ -24,6 +24,12 @@ discourse_activity_pub:
     default: 5
     min: 1
     client: true
+  activity_pub_post_status_visibility_groups:
+    default: "1|3|14" # auto group admin, staff, and trust_level_4
```

Comment:
I would prefer this to be publicly visible by default, interested instances can restrict this if they like.

Reply:
Done https://github.com/discourse/discourse-activity-pub/pull/82/commits/2857171719452646aebd596fea1819009015ed63